### ISSUE TITLE
*Still* not hot reloading

### ISSUE THREAD

**Issue by josephdburdick** [nozzle/react-static#45]
Hello, I'm unsure why my previous issue was closed. I understand changing data will not cause the app to reload but changing CSS, markup, etc should. This is not the case, even with starting with the latest `react-static@1.3.3`. Please advise.

**Comment by tannerlinsley** [comment]
Looking into this.

**Comment by coisnepe** [comment]
@tannerlinsley @josephdburdick Just created #50

**Comment by josephdburdick** [comment]
@coisnepe Interesting, maybe I force-quit an instance with terminal which had the process running? It didn't come to mind because afterwards I was able to use `create-react-app` to create a new app and test that hot reloading was working, and it did, on port 3000. 

Since reading the above notification and #50 I restarted my computer completely, went back and ran my `react-static` test again and it worked. I'll let one of the contributors close this issue out because perhaps there's an underlying issue where a lost process will prevent this project's future processes from hot reloading. Thanks for looking into this.
